core/src/main/java/org/geysermc/geyser/translator/protocol/bedrock/entity/player/input/BedrockPlayerAuthInputTranslator.java

```diff
@@ -78,6 +78,7 @@ public void translate(GeyserSession session, PlayerAuthInputPacket packet) {
         SessionPlayerEntity entity = session.getPlayerEntity();
 
         session.setClientTicks(packet.getTick());
+        session.setInputData(packet.getInputData());
```

Comment:
let's just store vertical_collision on the session player entity instead; both to avoid storing unnecessary data and removing variables stored in geysersession